libarchive/archive_read_support_filter_lzop.c

```diff
@@ -40,9 +40,6 @@ __FBSDID("$FreeBSD$");
 #ifdef HAVE_STRING_H
 #include <string.h>
 #endif
-#ifdef HAVE_UNISTD_H
-#include <unistd.h>
-#endif
```

Comment:
Removing headers must be done _very_ carefully.   Libarchive builds on a lot of different platforms, many of which have headers that behave quite differently.  _PLEASE_ make changes like this in a separate Pull Request.